### PR TITLE
ui avatar component created issue#74

### PR DESCRIPTION
**Pull request by gokhanap**

### CODE REVIEW

File: src/components/ui/avatar.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      { src
+        ? <UIImage className ={cls(style.img)} src={src} height={150} alt=""/>
+        : <p className ={cls(style.letter[size])}>{letters}</p>
+      }
+    </div>
+  );
+};
+
+Avatar.defaultProps = {
+  letters: 'KO',
```
Comment (SelmanKahya) [inline src/components/ui/avatar.js]: lets remove default letters

File: src/components/ui/avatar.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    xs: 'w-6 h-6',
+  },
+};
+
+const Avatar = (props) => {
+  const { size, src, letters } = props;
+  return (
+    <div className={cls(style.size[size], style.common)}>
+      { src
+        ? <UIImage className ={cls(style.img)} src={src} height={150} alt=""/>
```
[the remark this answers is not in the document]
Comment (gokhanap) [reply src/components/ui/avatar.js]: ok

File: src/components/ui/avatar.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    sm: 'w-8 h-8',
+    xl: 'w-48 h-48',
+    xs: 'w-6 h-6',
+  },
+};
+
+const Avatar = (props) => {
+  const { size, src, letters } = props;
+  return (
+    <div className={cls(style.size[size], style.common)}>
```
[the remark this answers is not in the document]
Comment (gokhanap) [reply src/components/ui/avatar.js]: ok

File: src/components/ui/avatar.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Avatar.defaultProps = {
+  letters: 'KO',
+  size: 'md',
+  src: null,
+};
+
+Avatar.propTypes = {
+  letters: PropTypes.string.isRequired,
+  size: PropTypes.oneOf(['xs', 'sm', 'md', 'lg', 'xl']).isRequired,
```
[the remark this answers is not in the document]
Comment (gokhanap) [reply src/components/ui/avatar.js]: ok

File: src/css/vendor/tailwind.setup.js [hunk 5]
```diff
@@ -391,6 +391,19 @@ module.exports = {
   */
 
   width: {
+    '1': '0.25rem',
```
[the remark this answers is not in the document]
Comment (gokhanap) [reply src/css/vendor/tailwind.setup.js]: ok

**Comment by gokhanap** [comment]
screen shot 2018-11-11 at 00 41 59


> looks good, left some comments. can you also share a browser screenshot of the component?

File: src/components/ui/avatar.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    sm: 'text-sm',
+    xl: 'text-3xl',
+    xs: 'text-xs',
+  },
+  size: {
+    lg: 'w-16 h-16',
+    md: 'w-12 h-12',
+    sm: 'w-8 h-8',
+    xl: 'w-48 h-48',
+    xs: 'w-6 h-6',
```
[the remark this answers is not in the document]
Comment (gokhanap) [reply src/components/ui/avatar.js]: cant inspect on Abstract too. try to match visually at least.

**Comment by gokhanap** [comment]
screen shot 2018-11-11 at 00 46 28

**Comment by SelmanKahya** [comment]
looks great, feel free to merge once CI job passes. we can add unit tests later.

**Comment by gokhanap** [comment]
test suite added

**Comment by SelmanKahya** [comment]
looks great

**Pull request merged**